### ISSUE TITLE
Remove parameter from getUser?

### ISSUE THREAD

**Issue by lorensr** [accounts-js/accounts#412]
`graphql-client.js`:
```
  public async getUser(accessToken: string): Promise<User> {
```

The server should be able to use the access token from the request header / get the user from `context.user`—it would be nicer/simpler if I didn't have to provide the parameter.

**Comment by pradel** [comment]
👍

**Comment by elie222** [comment]
Same would apply to `logout` I take it.

**Comment by elie222** [comment]
Made a PR for the fix (for getUser):
https://github.com/accounts-js/accounts/pull/420

May need to still change something around the tests for it.

**Comment by TimMikeladze** [comment]
Closed by #420 

Published `0.3.0-beta.25`
